### ISSUE TITLE
valid-describe with expect

### ISSUE THREAD

**Issue by benmonro** [jest-community/eslint-plugin-jest#342]
curious if we can/should modify valid-describe to report an error if it finds an expect in a describe.

for example:

```javascript
describe("some stuff", () => {
  expect("blah").toBe("foo");
}
```

jest won't report that as a test so wondering if it should be reported as an error.  if it should be reported should we put that in valid-describe or in a standalone rule called `no-standalone-expect`

thoughts?

**Comment by SimenB** [comment]
I like this idea. A separate rule throwing if an `expect` is outside of a test case might make sense? Potentially with a flag that just disallows it in hooks and `describe`s (so people can use helper functions for shared assertions), maybe? I don't think it needs to be coupled to `valid-describe`. _Could_ stick it in `valid-expect`, but I think that's already getting super advanced and a separate rule makes more sense. Maybe not, though 🙂
